apps/server/lib/bootstrap.ts

```diff
@@ -184,7 +139,7 @@ export const bootstrap = async (context, options) => {
 		try {
 			const key = await getActorKey(
 				context,
-				jellyfish,
+				jellyfish as any,
```

Comment:
We'd like to reduce in the long run `any`s instead of adding more. What would be required to give this the proper types?